rclcpp/include/rclcpp/subscription.hpp

```diff
@@ -361,6 +361,29 @@ class Subscription : public SubscriptionBase
     }
   }
 
+  // Take the loaned message
+  /*
+   * \param loaned_message : message that the user wants to take
+   */
+  bool
+  take_loaned_message(
+    void * loaned_message,
+    const rclcpp::MessageInfo & message_info) override
```

Comment:
```suggestion
    rclcpp::MessageInfo & message_info) override
```

`message_info` is lost in the doc.

Reply:
the doc is also missing information about the bool returned value and the fact that this function can throw